packages/app/components/Settings/ChatbotSettings.tsx

```diff
@@ -172,9 +239,10 @@ export default function ChatbotSettings(): ReactElement {
           headers: { HMS_API_TOKEN: profile.chat_token.token },
         })
 
-        toast.success('File uploaded.')
+        toast.success(`${file.name} uploaded.`)
```

Comment:
I realized that there is an error here. This call doesn't check status response and does toast.success even though a 500 server error is returned. 
It's not caused by your changes, but I wonder can you edit it?

Reply:
I see what you mean, I will update. so here's what I'll do, if upload for a given document fail, toast.failure("{doc1} upload failed") and go on with the rest of the process(delete doc2, doc3,...). let me know if you have any other suggestion

Reply:
Sounds good, and that's it